pilot/pkg/ambient/controller/waypoint_controller.go

```diff
@@ -127,7 +120,7 @@ func (rc *WaypointProxyController) Reconcile(name types.NamespacedName) error {
 		// Mostly used to avoid issues with local runs
 		return fmt.Errorf("injection config invalid, skipping reconile")
 	}
-	log := waypointLog.WithLabels("gateway", name.String())
+	scopedLog := waypointLog.WithLabels("gateway", name.String())
```

Comment:
replacing `log` pattern is intentionally used here (and elsewhere) to ensure folks use the correctly scoped log and don't accidentally fall back to the unscoped one